SimpleTDDCsharpTest/PalindromeCheckerTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿
+
```

Comment:
why extra space on top?

Reply:
by mistake i will make sure it will not happen again